src/components/research/resizable-graph.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import useWindowSize from "../../util/useWindowSize";
+import { BreakPoint, screenSizes, mq } from "../../util/mq";
+import { css } from "@emotion/core";
+
+const graphContainerStyles = (fullSpan: boolean) => css`
+  ${mq(BreakPoint.lg)} {
+    ${fullSpan ? "grid-column: 1 / span  2" : ""};
+  }
+`;
+
+interface ResizableGraphProps {
+  mobile: React.ReactNode;
+  tablet: React.ReactNode;
+  desktop: React.ReactNode;
+  children: any;
+  fullSpan: boolean;
+}
+
+const ResizableGraph = ({
+  mobile,
+  tablet,
+  desktop,
+  fullSpan,
+  children
+}: ResizableGraphProps) => {
+  const { width } = useWindowSize();
+
+  let Graph = undefined;
+
+  if (width < screenSizes[BreakPoint.md]) {
+    Graph = mobile;
+  } else if (width < screenSizes[BreakPoint.lg]) {
+    Graph = tablet;
+  } else {
+    Graph = desktop;
+  }
```

Comment:
We can avoid this `useWindowSize()` stuff by loading the svg into an [`<img>` with a `srcset` attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#How_do_you_create_responsive_images).

There's also a gatsby-image way to do this, which I used in `/components/operational-review/relief-image.tsx`.